builder/builder-next/builder.go

```diff
@@ -558,6 +570,18 @@ func toBuildkitExtraHosts(inp []string) (string, error) {
 	return strings.Join(hosts, ","), nil
 }
 
+// toBuildkitUlimits converts ulimits from docker type=soft:hard format to buildkit's csv format
+func toBuildkitUlimits(inp []*units.Ulimit) (string, error) {
```

Comment:
oh... hmm... so actually looks like this never return an error? ok to fix later (somewhat surprised there's no longer complaining)